notebooks/00_quick_start/xdeepfm_single_node.ipynb

```diff
@@ -0,0 +1,405 @@
+      "at epoch 7 , train time: 4.1 eval time: 3.9\n",
+      "at epoch 8 train info: auc:0.964, logloss:0.2523 eval info: auc:0.9133, logloss:0.3772\n",
+      "at epoch 8 , train time: 4.1 eval time: 4.0\n",
+      "at epoch 9 train info: auc:0.9799, logloss:0.1921 eval info: auc:0.9346, logloss:0.3318\n",
+      "at epoch 9 , train time: 4.0 eval time: 4.0\n",
+      "at epoch 10 train info: auc:0.9897, logloss:0.1411 eval info: auc:0.9515, logloss:0.2884\n",
+      "at epoch 10 , train time: 4.0 eval time: 4.0\n",
+      "at epoch 11 train info: auc:0.9952, logloss:0.1004 eval info: auc:0.9641, logloss:0.2499\n",
+      "at epoch 11 , train time: 4.0 eval time: 4.1\n",
+      "at epoch 12 train info: auc:0.998, logloss:0.0696 eval info: auc:0.9729, logloss:0.2189\n",
+      "at epoch 12 , train time: 4.0 eval time: 4.0\n",
+      "at epoch 13 train info: auc:0.9993, logloss:0.0473 eval info: auc:0.9787, logloss:0.196\n",
+      "at epoch 13 , train time: 4.0 eval time: 4.0\n",
+      "at epoch 14 train info: auc:0.9998, logloss:0.0317 eval info: auc:0.9825, logloss:0.1801\n",
+      "at epoch 14 , train time: 4.0 eval time: 4.0\n",
+      "at epoch 15 train info: auc:0.9999, logloss:0.0212 eval info: auc:0.9851, logloss:0.1693\n",
+      "at epoch 15 , train time: 4.2 eval time: 4.0\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<reco_utils.recommender.deeprec.models.xDeepFM.XDeepFMModel at 0x2451c0c9278>"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.fit(train_file, valid_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Again, let's see what is the model's performance now (after training):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'auc': 0.9851, 'logloss': 0.1693}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(model.run_eval(valid_file))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we want to get the full prediction scores rather than evaluation metrics, we can do this:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<reco_utils.recommender.deeprec.models.xDeepFM.XDeepFMModel at 0x2451c0c9278>"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.predict(test_file, output_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we have successfully launched an experiment on a synthetic dataset. Next let's try something on a real world dataset, which is a small sample from Criteo dataset \\[2\\]."
```

Comment:
I think we should explain a little bit what is Criteo, size, data in it, etc.

Reply:
Sure. Done.